packages/asset/contracts/Asset.sol

```diff
@@ -354,4 +354,6 @@ contract Asset is
         require(registry != address(0), "Asset: registry can't be zero address");
         operatorFilterRegistry = IOperatorFilterRegistry(registry);
     }
+
+    uint256[50] private __gap;
```

Comment:
Should be [49] because of mapping on line 52?

Reply:
Right, I did not count mappings to occupy slots, thanks for the documentation.
I will make an upate.